src/sentry/api/endpoints/organization_discover_query.py

```diff
@@ -129,6 +129,15 @@ def validate_conditions(self, attrs, source):
             attrs[source] = conditions
         return attrs
 
+    def validate_aggregations(self, attrs, source):
+        valid_functions = set(['count()', 'uniq', 'avg'])
+        requested_functions = set(agg[0] for agg in attrs[source])
+
+        if not requested_functions.issubset(valid_functions):
+            raise serializers.ValidationError('Invalid aggregate function')
```

Comment:
might be nice to include the invalid one in the message (like `Invalid aggregate function: "sum"`)